MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,748 @@
+---
+layout: post
+title: Layout in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the Layout support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Layout in Xamarin DataForm (SfDataForm)
+
+## Overview
+
+The data form supports linear and grid layouts. The DataFormLayoutManager creates the [DataFormItemView] and manages layout of label, editor, and validation label.
+
+## Linear layout support
+
+By default, the data form arranges the fields one-by-one. It is applicable for both label positions: left and top.
+
+When the label position is Left, the linear layout is shown as follows:
+
+When the label position is Top, the linear layout is shown as follows:
+
+## Grid layout support
+
+By default, the data form arranges one data field per row. It is possible to have more than one data fields per row by setting the [ColumnCount] property which provides grid like layout for the data form.
```

Comment:
Mention about ItemsOrderInRow for Grid layout

Reply:
Mentioned about ItemsOrderInRow for Grid layout